### DISCUSSION TITLE
keyboard shortcut for forward/ backward button

### DISCUSSION THREAD

**Discussion by overcesium** [keiwando/toonsquid#165] [Ideas]
Hello, I'm a professional traditional 2d animator who discovered Toon Squid today. What you did with this software is unbelievable and a true game-changer. After laying for a couple of hours, I found myself at home, being a daily Toon Boom and Adobe Animate user.

I have a suggestions for future updates:
You should add a keyboard shortcut for the forward/ backward buttons. The current "," and "." are not very convenient. As a trad animator, I prefer to quickly flip through drawings instead of through the frames, but by default, I can access the flipping through the drawings only by touch screen.

Thank you so much,
I can finally work away fro the desk!

**Comment by keiwando** [comment]
Since the 1.9 update you can hold shift while pressing the `,` and `.` buttons in order to flip through drawings instead of frames.

**Reply by andydjames** [reply]
Would it be possible to add two other keyboard shortcuts to move to next keyframe and previous keyframe Preferrably a single button press rather than SHift + or Cmd + for one finger operation.
Many thanks.
